source/concurrent/native/ConditionWin32.ooc

```diff
@@ -45,7 +37,7 @@ ConditionWin32: class extends WaitCondition {
 		this _waitingEvents add(eventId)
 		this _mutex unlock()
 		mutex unlock()
-		(WaitForSingleObject(eventId, Infinite) == WaitSuccess)
+		(WaitForSingleObject(eventId, INFINITE) == WaitSuccess)
```

Comment:
If it is called  `INFINITE`on all other places, I guess it is easier to change this one only instead of all the others.